tests/app/conftest.py

```diff
@@ -289,6 +289,22 @@ def sample_job_with_placeholdered_template(
 
 
 @pytest.fixture(scope='function')
+def sample_scheduled_job(
+    notify_db,
+    notify_db_session,
+    service=None
```

Comment:
should this be `sample_service`, so it can be used without defining a service separately?